src/main/kotlin/com/cognifide/gradle/common/file/transfer/FileTransferManager.kt

```diff
@@ -32,21 +32,21 @@ class FileTransferManager(private val common: CommonExtension) : FileTransfer {
         common.prop.string("fileTransfer.password")?.let { set(it) }
     }
 
+    var bearerToken: String? = null
```

Comment:
common.obj.string as for other mutable props;


see https://docs.gradle.org/current/userguide/lazy_configuration.html

Reply:
didn't occur to me to leave "string { }" empty, Fixed, Thanks!